zerver/lib/email_notifications.py

```diff
@@ -206,8 +206,14 @@ def build_message_list(
     messages_to_render: list[dict[str, Any]] = []
 
     def sender_string(message: Message) -> str:
-        if message.recipient.type in (Recipient.STREAM, Recipient.DIRECT_MESSAGE_GROUP):
+        if message.recipient.type == Recipient.STREAM:
             return message.sender.full_name
+        elif message.recipient.type == Recipient.DIRECT_MESSAGE_GROUP:
+            display_recipient = get_display_recipient(message.recipient)
+            if len(display_recipient) > 2:
+                return message.sender.full_name
+            else:
+                return ""
         else:
```

Comment:
The `else --> return` can be simplified to just `return` :)

Reply:
Good call. Updated.